### PR TITLE
Add the initial metrics API definition

### PR DESCRIPTION
**Pull request by vmarkovtsev**
Signed-off-by: Vadim Markovtsev <vadim@athenian.co>

### CODE REVIEW

File: spec/openapi.yaml [hunk 1]
```diff
@@ -0,0 +1,108 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Athenian Dashboard API
+  license:
+    name: CC-BY-4.0
```
Comment (dpordomingo) [inline spec/openapi.yaml]: I think that `CC` is not a proper software license. On the other side, I know that @se7entyse7en already left this as something to be defined later, so he preferred to leave it empty (which use to mean: 'all rights reserved')

Comment (vmarkovtsev) [reply spec/openapi.yaml]: This is the license of the API document (that's stated in the spec!), and it was the most popular according to https://matija.suklje.name/popular-licenses-in-openapi
So no worries.

**Comment by vmarkovtsev** [comment]
@dpordomingo Regarding the endpoints: they are listed in this doc.
However, I need to add a few service ones to let you consume `/metrics`.

**Comment by vmarkovtsev** [comment]
BTW: here is the link to the spec that guides me: https://github.com/OAI/OpenAPI-Specification/blob/master/versions/3.0.0.md

File: spec/openapi.yaml (hunk 1 above)
Comment (vmarkovtsev) [reply spec/openapi.yaml]: @eiso says: no licenses. So changed to proprietary (default).

Comment (se7entyse7en) [reply spec/openapi.yaml]: Is this a real issue? I mean the license of the spec? I just don't know whether there having a CC-BY license for the spec is in conflict with being part of an unlicensed project. \cc @eiso

**Pull request merged**